src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -75,6 +75,9 @@
     <Reference Include="PresentationFramework" />
     <Reference Include="ProtoGeometry, Version=2.12.0.5595, Culture=neutral, processorArchitecture=AMD64">
       <HintPath>..\packages\DynamoVisualProgramming.LibG_227_0_0.2.12.0.5595\lib\net48\ProtoGeometry.dll</HintPath>
+    </Reference>
+    <Reference Include="ProtoGeometry, Version=2.12.0.5264, Culture=neutral, processorArchitecture=AMD64">
```

Comment:
Can you delete this reference, it is duplicate with the one above

Reply:
Done.